### ISSUE TITLE
Bot fails after choosing shipping address.

### ISSUE THREAD

**Issue by VIncentOOF** [Hari-Nagarajan/fairgame#807]
After I used --shipping bypass and the bot finally selected my address, at the deliver area it failed to press continue.
Is it also possible to click shipping that is not free
![fail2](https://user-images.githubusercontent.com/85739638/123896825-e5e04700-d994-11eb-9592-b76267adaf84.jpg)
![fail1](https://user-images.githubusercontent.com/85739638/123896836-e973ce00-d994-11eb-90ae-1899857ba8a5.jpg)

**Comment by DakkJaniels** [comment]
There is no code included to deal with selecting delivery options.

**Comment by VIncentOOF** [comment]
So how do I fix this issue because I am ok with the bot selecting any type of shipping and don’t care what the bot selects also how is the bot coded for shipping

**Comment by DakkJaniels** [comment]
You add code to the program to click the button you want to click.

**Comment by VIncentOOF** [comment]
But is there any default payment and shipping code like from which line to which line so that I can maybe use that to guide me

**Comment by DakkJaniels** [comment]
No, because it is expected/assumed that those are already set and you don't have to select them every time. This does not typically come up for domestic shipping.

**Comment by ChristianPerdomo** [comment]
The address is set by setting the "prime days" option as default and this is how the bot works

**Comment by VIncentOOF** [comment]
Ah so only local shipping works? And I have Amazon prime for Amazon Sg so I will just buy prime for Amazon US and use my planet express address in the USA.

**Comment by xiurobert** [comment]
Well, the bot is just using Selenium. Find the XPath of the shipping options and use the bot to click on it.
